src/Hl7.Fhir.Specification/Specification/Snapshot/SnapshotGenerator.cs

```diff
@@ -1702,7 +1702,17 @@ private void addSliceBase(ElementDefinitionNavigator snap, ElementDefinitionNavi
 
             result = snap.ReturnToBookmark(lastSlice);
             // Copy the original (unmerged) slice base element to snapshot
-            if (result) { result = snap.InsertAfter((ElementDefinition)sliceBase.Current.DeepCopy()); }
+            if (result)
```

Comment:
I think we need to fix this in an earlier stage (when merging probably) to also tackle the case where you just remove the bindable types without slicing.

Reply:
Thanks for the unittest which proves this